packages/lodestar-cli/src/cmds/beacon/testnets/altona.ts

```diff
@@ -1,50 +1,34 @@
+import {IBeaconNodeOptionsPartial} from "../../../options";
+
 /* eslint-disable max-len */
 
 // Use a Typescript file instead of JSON so it's automatically included in the built files
 
-export const altonaConfig = {
-  "api": {
-    "enabled": true,
-    "namespaces": ["beacon", "validator"],
```

Comment:
not sure where are default values, but "node" should be included here

Reply:
Defaults can be found here
https://github.com/ChainSafe/lodestar/blob/c382969c82ddc821dfc899368ac1efd5ebb8ac62/packages/lodestar/src/api/rest/options.ts#L13